Libraries/LibWeb/WebAssembly/Memory.idl

```diff
@@ -1,6 +1,8 @@
 dictionary MemoryDescriptor {
     required [EnforceRange] unsigned long initial;
     [EnforceRange] unsigned long maximum;
+    // https://github.com/WebAssembly/threads/blob/main/proposals/threads/Overview.md#webassemblymemory-constructor
+    boolean shared;
```

Comment:
here, we want to put a ref to the updated IDL

https://webassembly.github.io/threads/js-api/index.html#dictdef-memorydescriptor

The boolean should be default false.

Reply:
Thanks! Fixed.